lib/ace/ext/katex.js

```diff
@@ -1,10 +1,5 @@
 define(function(require, exports, module) {
     var katex = require("ace/ext/katex/katex");
-    var cssKatexPath = require.toUrl("./katex/katex.min.css");
-    var linkKatex = $("<link>").attr({
-      rel: "stylesheet",
-      href: cssKatexPath
-    });
-    $("head").append(linkKatex);
-    module.exports = katex;
+    window.loadCss("katex");
```

Comment:
This breaks the katex demo: `window.loadCss` is not a function. I suggest wrapping code which used to be here into a function and defining it somewhere in demo page